torchvision/extension.py

```diff
@@ -23,10 +27,25 @@ def _register_extensions():
 try:
     _register_extensions()
     _HAS_OPS = True
+    def _has_ops():
+        return True
 except (ImportError, OSError):
     pass
```

Comment:
Thanks for addressing this!
Perhaps it's possible to define `_has_ops` function here that returns `(bool has_ops, str err_msg)`. Then the exact error message from the exception can be used in `_assert_has_ops`

Reply:
Thanks for the review, I forgot to answer this.

I think this should be possible to be done, let me try this.